.github/workflows/wix.yaml

```diff
@@ -34,6 +36,59 @@ jobs:
     - run: npm run build -- --win --publish=never
       env:
         RD_FEAT_WIX: '1'
+    - name: Upload Windows installer
+      uses: actions/upload-artifact@v3
+      with:
+        name: Rancher Desktop Setup unsigned.msi
+        path: dist/Rancher Desktop*.msi
+        if-no-files-found: error
+    - name: Upload Windows zip
+      uses: actions/upload-artifact@v3
+      with:
+        name: Rancher Desktop-win.zip
+        path: dist/Rancher Desktop-*-win.zip
+        if-no-files-found: error
+  sign:
+    name: Test Signing
+    needs: package
+    runs-on: windows-latest
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        persist-credentials: false
+    - name: Install Windows dependencies
+      shell: powershell
+      run: .\scripts\windows-setup.ps1 -SkipVisualStudio -SkipTools
+    - uses: actions/setup-go@v3
+      with:
+        go-version: '^1.18'
+    - run: npm ci
+    - uses: actions/download-artifact@v3
+      with:
+        name: Rancher Desktop-win.zip
+    - shell: powershell
+      env:
+        RD_FEAT_WIX: '1'
+      run: |
+        # Generate a test signing certificate
+        $cert = New-SelfSignedCertificate `
+          -Type Custom `
+          -Subject "CN=Rancher-Sandbox, C=CA" `
```

Comment:
Should the `CN=Rancher-Sandbox` but FQDN? e.g. `CN=rancher-sandbox/rancher-desktop`

Reply:
No, because this is a code-signing certificate, and not used to identify a host (as in a machine with a host name).  For example, Microsoft uses one with `CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmon, S=Washington, C=US`.